Add `osu!taiko World Cup 2026: Semifinals Recap` news post (#14624)
ppy/osu-wiki: news/2026/2026-04-18-osutaiko-world-cup-2026-semifinals-recap.md

The show goes on! The few remaining teams put up a great show for the crowds. Exciting matches and a lot of fun during last week's Semifinals. Despite all the pressure and high stakes, every team has been performing at the top level! As opponents and mappools get tougher, players keep having only one thing in mind: Grabbing that crown and taking it home.

Read on to find out everything you need to know about what happened last weekend!

## Mappool showcase

**[Download the mappool here! (99 MB)](https://packs.ppy.sh/P319%20-%20osu%21taiko%20World%20Cup%202026%3A%20Semifinals.zip)**

<div align="center" class="osu-md__paragraph">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?autoplay=false&parent=osu.ppy.sh&video=2740668428" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
</div>

![Semifinals mappool table](/wiki/shared/news/2026-04-18-osutaiko-world-cup-2026-semifinals-recap/mappool-sf.jpg)

## osu! originals

<div align="center" class="osu-md__paragraph">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/Kzi-trSYpnY" frameborder="0" allowfullscreen></iframe>
</div>

> *crrzzzz… shhhk… krrrttt…* That isn't just intereference. Out of this noise emerges [SIGNALBURNERRR](https://osu.ppy.sh/beatmapsets/2534365#taiko/5604141), the newest addition to the osu!taiko World Cup 2026 osu! originals lineup. Produced by [ALEPH](https://osu.ppy.sh/beatmaps/artists/107), known for their many unique and creative compositions, this track is no exception. A great example of the artist's ability to create a technical and complicated sound, and therefore a great fit for the Semifinals' NM6 slot. Fix your antenna and enjoy this week's osu! original: **SIGNALBURNERRR**.

Reviewer: ```suggestion
*crrzzzz… shhhk… krrrttt…* That isn't just interference. Out of this noise emerges [SIGNALBURNERRR](https://osu.ppy.sh/beatmapsets/2534365#taiko/5604141), the newest addition to the osu!taiko World Cup 2026 osu! originals lineup. Produced by [ALEPH](https://osu.ppy.sh/beatmaps/artists/107), known for their many unique and creative compositions, this track is no exception. A great example of the artist's ability to create a technical and complicated sound, and therefore a great fit for the Semifinals' NM6 slot. Fix your antenna and enjoy this week's osu! original: **SIGNALBURNERRR**.
```

| 39 | **Japan** ::{ flag=JP }:: | **6** | 1 | ::{ flag=KR }:: South Korea | [#1](https://osu.ppy.sh/community/matches/120916660) | [#1](https://www.twitch.tv/videos/2745512622) |
| 34 | **United States** ::{ flag=US }:: | **6** | 1 | ::{ flag=FI }:: Finland | [#1](https://osu.ppy.sh/community/matches/120918218) | [#1](https://www.twitch.tv/videos/2745562049) |
| 36 | **Italy** ::{ flag=IT }:: | **6** | 4 | ::{ flag=GB }:: United Kingdom | [#1](https://osu.ppy.sh/community/matches/120919997) | [#1](https://www.twitch.tv/videos/2745734198) |
| 40 | **Chile** ::{ flag=CL }:: | **6** | 3 | ::{ flag=BR }:: Brazil | [#1](https://osu.ppy.sh/community/matches/120922024) | [#1](https://www.twitch.tv/videos/2745993870) |
| 37a | United States ::{ flag=US }:: | 3 | **6** | ::{ flag=TW }:: **Taiwan** | [#1](https://osu.ppy.sh/community/matches/120922696) | [#1](https://www.twitch.tv/videos/2746035528) |
| 38a | Italy ::{ flag=IT }:: | 5 | **6** | ::{ flag=CN }:: **China** | [#1](https://osu.ppy.sh/community/matches/120924693) | [#1](https://www.twitch.tv/videos/2746243641) |

> Great efforts from every team but unfortunately, some brave teams were not able to beat their opponents and advance to the Finals. A massive shoutout to the following teams for their amazing performance up to this point and we look forward to seeing them again next year: ::{ flag=FI }:: Finland, ::{ flag=IT }:: Italy, ::{ flag=US }:: United States, ::{ flag=AU }:: Australia, ::{ flag=NZ }:: New Zealand and ::{ flag=GB }:: United Kingdom.

Reviewer: feels a bit lukewarm to read, how about:

```suggestion
Great efforts from every team but unfortunately, some teams saw their journeys end just before the Finals. A massive shoutout to the following teams for their outstanding performance so far — we look forward to seeing them again next year: ::{ flag=FI }:: Finland, ::{ flag=IT }:: Italy, ::{ flag=US }:: United States, ::{ flag=AU }:: Australia, ::{ flag=NZ }:: New Zealand and ::{ flag=GB }:: United Kingdom.
```

> Further detailed statistics for this round can be found [here](https://docs.google.com/spreadsheets/d/1jfo9ZsZN2hBsc9Av1JGodXUZnj_6hru9PT2JZ_Feeq4).

Reviewer: nitpick but longer hypertext is easier to notice
```suggestion
Further detailed statistics for this round can be found [in this page](https://docs.google.com/spreadsheets/d/1jfo9ZsZN2hBsc9Av1JGodXUZnj_6hru9PT2JZ_Feeq4).
```

## Match highlights

### ::{ flag=IT }:: Italy vs. ::{ flag=CN }:: China

**[View the match information here!](https://osu.ppy.sh/community/matches/120924693)**

<div align="center" class="osu-md__paragraph">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?autoplay=false&parent=osu.ppy.sh&video=2746243641" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
</div>

> TWC 2026's lower bracket has been producing classics for weeks now. But on Sunday, a pair of teams met in what may be the match of the tournament. Italy, the **last European team** standing and fresh off a 6-4 win against the United Kingdom, faced longtime powerhouse China after their 6-0 win against Australia, the last spot in Finals being contested by seeds 6 and 7. Don't say it, don't think it…

Reviewer 2: very funny yes

```suggestion
TWC 2026's lower bracket has been producing classics for weeks now. But on Sunday, a pair of teams met in what may be the match of the tournament. Italy, the **last European team** standing and fresh off a 6–4 win against the United Kingdom, faced longtime powerhouse China after their 6–0 win against Australia, the last spot in Finals being contested by adjacent seeds.
```

> TWC 2026's lower bracket has been producing classics for weeks now. But on Sunday, a pair of teams met to produce what may be the match of the tournament. The **last European team** standing in Italy, fresh off a 6-4 win against the United Kingdom, faced longtime powerhouse China after their 6-0 win against Australia, the last spot in Finals being contested by seeds 6 and 7. Don't say it, don't think it…

Reviewer: less producing, also `The **last European team** standing in Italy` doesn't make a lot of sense, would suggest:
```suggestion
TWC 2026's lower bracket has been producing classics for weeks now. But on Sunday, a pair of teams met in what may be the match of the tournament. Italy, the **last European team** standing and fresh off a 6-4 win against the United Kingdom, faced longtime powerhouse China after their 6-0 win against Australia, the last spot in Finals being contested by seeds 6 and 7. Don't say it, don't think it…
```

> China won the roll, and led with a strong pick of [pa-o-mu99999 - Bobobo-bo bo-bobo](https://osu.ppy.sh/beatmapsets/2534673#taiko/5604911) (NM4), holding their pick with ease. Italy proceeded to pick [Xyris - Terrablazer](https://osu.ppy.sh/beatmapsets/2534374#taiko/5604173) (HD2) and blow away every other score of the weekend, including China and even some winner's bracket teams, with a combined 1x100 double FC! China's next pick of [Skybreak - NOLIGHT (fear. HeyBela)](https://osu.ppy.sh/beatmapsets/2534739#taiko/5605025) (NM2) was the shakiest so far, with Italy spending the entire first half in front before China came back to win by just over 20,000. Italy picked [MetaHumanBoi - Illustrious Drifters](https://osu.ppy.sh/beatmapsets/2534393#taiko/5604205) (FM1) and gained the mod advantage with two HDHR plays. That mod advantage turned out crucial, as Italy won despite only one FC to China's two! Now at a score of 2-2, China continued their spree of tech-based picks by holding on through [Camellia - Maze of Vignere Square](https://osu.ppy.sh/beatmapsets/2534390#taiko/5604200) (DT1), before Italy picked [Toromaru - Erinyes](https://osu.ppy.sh/beatmapsets/2534384#taiko/5604192) (HR1), where we would learn that Italy's [ndrrr](https://osu.ppy.sh/users/4609767) was having a very, very good day.

Reviewer 2: ```suggestion
China won the roll, and led with a strong pick of [pa-o-mu99999 - Bobobo-bo bo-bobo](https://osu.ppy.sh/beatmapsets/2534673#taiko/5604911) (NM4), holding their pick with ease. Italy proceeded to pick [Xyris - Terrablazer](https://osu.ppy.sh/beatmapsets/2534374#taiko/5604173) (HD2) and blow away every other score of the weekend, including China and even some winner's bracket teams, with a combined 1x100 double FC! China's next pick of [Skybreak - NOLIGHT (fear. HeyBela)](https://osu.ppy.sh/beatmapsets/2534739#taiko/5605025) (NM2) was the shakiest so far, with Italy spending the entire first half in front before China came back to win by just over 20,000. Italy picked [MetaHumanBoi - Illustrious Drifters](https://osu.ppy.sh/beatmapsets/2534393#taiko/5604205) (FM1) and gained the mod advantage with two HDHR plays.

That mod advantage turned out crucial, as Italy won despite their only one FC to China's two! Now at a score of 2–2, China continued their spree of tech-based picks by holding on through [Camellia - Maze of Vignere Square](https://osu.ppy.sh/beatmapsets/2534390#taiko/5604200) (DT1), before Italy picked [Toromaru - Erinyes](https://osu.ppy.sh/beatmapsets/2534384#taiko/5604192) (HR1), where we would learn that Italy's [ndrrr](https://osu.ppy.sh/users/4609767) was having a very, very good day.
```

> Despite a rare bit of shaky play from Italy's star player and **the strongest player in Europe** ::{ flag=IT }:: [A-40](https://osu.ppy.sh/users/14510301), Italy won this map by 5,890 on the back of ::{ flag=IT }:: [ndrrr](https://osu.ppy.sh/users/4609767)'s 99.47% FC, sitting atop the lobby by over 30,000 score and clutching another held pick for his team. After China won [Riya - IMPULSE](https://osu.ppy.sh/beatmapsets/2534380#taiko/5604182) (HD1) by a huge margin of over 250,000, Italy went back to the HR well, choosing [MITCH DOWNVELL - not sakura](https://osu.ppy.sh/beatmapsets/2534381#taiko/5604184) (HR2). Due to Italy's sheer strength on HR accuracy, ::{ flag=CN }:: [kknegative](https://osu.ppy.sh/users/2349769) and ::{ flag=CN }:: [WLYMinato](https://osu.ppy.sh/users/12703319)'s double FC wasn't enough despite a pair of Italian misses, and the match reached 4-4, essentially becoming a best of 3 with a ticket to the Finals on the line. Each team would get to pick just one more map.

Reviewer 2: ```suggestion
Despite a rare bit of shaky play from Italy's star player and **the strongest player in Europe** ::{ flag=IT }:: [A-40](https://osu.ppy.sh/users/14510301), Italy won this map by 5,890 on the back of ::{ flag=IT }:: [ndrrr](https://osu.ppy.sh/users/4609767)'s 99.47% FC, sitting atop the lobby by over 30,000 score and clutching another held pick for his team. After China won [Riya - IMPULSE](https://osu.ppy.sh/beatmapsets/2534380#taiko/5604182) (HD1) by a huge margin of over 250,000, Italy went back to the HR well, choosing [MITCH DOWNVELL - not sakura](https://osu.ppy.sh/beatmapsets/2534381#taiko/5604184) (HR2). Due to Italy's sheer strength on HR accuracy, ::{ flag=CN }:: [kknegative](https://osu.ppy.sh/users/2349769) and ::{ flag=CN }:: [WLYMinato](https://osu.ppy.sh/users/12703319)'s double FC wasn't enough despite a pair of Italian misses, and the match reached 4–4, essentially becoming a best-of-3 with a ticket to the Finals on the line. Each team would get to pick just one more map.
```

> Despite a rare bit of shaky play from Italy's star player and **the strongest player in Europe** in ::{ flag=IT }:: [A-40](https://osu.ppy.sh/users/14510301), Italy won this map by 5,890 on the back of ::{ flag=IT }:: [ndrrr](https://osu.ppy.sh/users/4609767)'s 99.47% FC, sitting atop the lobby by over 30,000 score and clutching another held pick for his team. After China won [Riya - IMPULSE](https://osu.ppy.sh/beatmapsets/2534380#taiko/5604182) (HD1) by a huge margin of over 250,000, Italy went back to the HR well, choosing [MITCH DOWNVELL - not sakura](https://osu.ppy.sh/beatmapsets/2534381#taiko/5604184) (HR2). Due to Italy's sheer strength on HR accuracy, ::{ flag=CN }:: [kknegative](https://osu.ppy.sh/users/2349769) and ::{ flag=CN }:: [WLYMinato](https://osu.ppy.sh/users/12703319)'s double FC wasn't enough despite a pair of Italian misses, and the match reached 4-4, essentially becoming a best of 3 with a ticket to the Finals on the line. Each team would get to pick just one more map.

Reviewer: ```suggestion
Despite a rare bit of shaky play from Italy's star player and **the strongest player in Europe** ::{ flag=IT }:: [A-40](https://osu.ppy.sh/users/14510301), Italy won this map by 5,890 on the back of ::{ flag=IT }:: [ndrrr](https://osu.ppy.sh/users/4609767)'s 99.47% FC, sitting atop the lobby by over 30,000 score and clutching another held pick for his team. After China won [Riya - IMPULSE](https://osu.ppy.sh/beatmapsets/2534380#taiko/5604182) (HD1) by a huge margin of over 250,000, Italy went back to the HR well, choosing [MITCH DOWNVELL - not sakura](https://osu.ppy.sh/beatmapsets/2534381#taiko/5604184) (HR2). Due to Italy's sheer strength on HR accuracy, ::{ flag=CN }:: [kknegative](https://osu.ppy.sh/users/2349769) and ::{ flag=CN }:: [WLYMinato](https://osu.ppy.sh/users/12703319)'s double FC wasn't enough despite a pair of Italian misses, and the match reached 4-4, essentially becoming a best of 3 with a ticket to the Finals on the line. Each team would get to pick just one more map.
```

> China's choice was [BlackY - ULT!MATE END](https://osu.ppy.sh/beatmapsets/2534370#taiko/5604161) (NM3), opting for the speed nomod against a relatively weak high-speed team in Italy. This was also the only map they could use to get the dominant ::{ flag=IT }:: [A-40](https://osu.ppy.sh/users/14510301) out of the lobby, and forcing ::{ flag=IT }:: [ndrrr](https://osu.ppy.sh/users/4609767) to play speed was a good way to get a low score out of Italy… on any other day, that might have been true.

Reviewer 2: ```suggestion
China's choice was [BlackY - ULT!MATE END](https://osu.ppy.sh/beatmapsets/2534370#taiko/5604161) (NM3), opting for the speed No Mod pick against a relatively weak high-speed team in Italy. This was also the only map they could use to get the dominant ::{ flag=IT }:: [A-40](https://osu.ppy.sh/users/14510301) out of the lobby, and forcing ::{ flag=IT }:: [ndrrr](https://osu.ppy.sh/users/4609767) to play speed was a good way to get a low score out of Italy... on any other day, that might have been true.
```

> China's choice was [BlackY - ULT!MATE END](https://osu.ppy.sh/beatmapsets/2534370#taiko/5604161) (NM3), the speed nomod against a relatively weak high-speed team in Italy. This was also the only map they could use to get the dominant ::{ flag=IT }:: [A-40](https://osu.ppy.sh/users/14510301) out of the lobby, and forcing ::{ flag=IT }:: [ndrrr](https://osu.ppy.sh/users/4609767) to play speed was a good way to get a low score out of Italy… on any other day, that might have been true.

Reviewer 3: ```suggestion
China's choice was [BlackY - ULT!MATE END](https://osu.ppy.sh/beatmapsets/2534370#taiko/5604161) (NM3), opting for the speed nomod against a relatively weak high-speed team in Italy. This was also the only map they could use to get the dominant ::{ flag=IT }:: [A-40](https://osu.ppy.sh/users/14510301) out of the lobby, and forcing ::{ flag=IT }:: [ndrrr](https://osu.ppy.sh/users/4609767) to play speed was a good way to get a low score out of Italy… on any other day, that might have been true.
```

<div align="center" class="osu-md__paragraph">
    <video width="95%" controls>
        <source src="https://assets.ppy.sh/media/news/2026-04-18-twc26-sf-highlight-1.mp4" type="video/mp4" preload="none">
    </video>
</div>

> With an incredible performance, completely defying what people expected of his skillset, ::{ flag=IT }:: [ndrrr](https://osu.ppy.sh/users/4609767) blew through NM3 to bring his team exactly one chance at ending it before a tiebreaker. Italy would call a 2-minute timeout before picking [seatrus - TEMP3ST](https://osu.ppy.sh/beatmapsets/2534351#taiko/5604104) (NM1). This was the map Italy destroyed against the UK to claim their lone break point on Saturday, with a first-match score of 1,980,510, enough to beat China's Saturday performance by over 50,000 and secure victory, but this wasn’t Saturday anymore.

Reviewer 3: ```suggestion
With an incredible performance, completely defying what people expected of his skillset, ::{ flag=IT }:: [ndrrr](https://osu.ppy.sh/users/4609767) blew through NM3 to bring his team exactly one chance at ending it before a tiebreaker. Italy would call a 2-minute timeout before picking [seatrus - TEMP3ST](https://osu.ppy.sh/beatmapsets/2534351#taiko/5604104) (NM1). This was the map Italy destroyed against the UK to claim their lone break point on Saturday, with a first-match score of 1,980,510 that would've easily been enough to beat China's Saturday performance by over 50,000 and secure victory, but this wasn’t Saturday anymore.
```

Reviewer 3: yeah initial wording on this was pretty weird now i think of it

> With an incredible performance, completely defying what people expected of his skillset, ::{ flag=IT }:: [ndrrr](https://osu.ppy.sh/users/4609767) blew through NM3 to bring his team exactly one chance at ending it before a tiebreaker. Italy would call a 2-minute timeout before picking [seatrus - TEMP3ST](https://osu.ppy.sh/beatmapsets/2534351#taiko/5604104) (NM1). This was the map Italy destroyed against the UK to claim their lone break point on Saturday, with a first-match score of 1,980,510 that would've easily been enough to beat China's Saturday performance by over 50,000 and secure victory, but this wasn’t Saturday anymore.

Reviewer 2: ```suggestion
With an incredible performance, completely defying what people expected of his skill set, ::{ flag=IT }:: [ndrrr](https://osu.ppy.sh/users/4609767) blew through NM3 to bring his team exactly one chance at ending it before a tiebreaker. Italy called a 2-minute timeout before picking [seatrus - TEMP3ST](https://osu.ppy.sh/beatmapsets/2534351#taiko/5604104) (NM1). This was the map Italy destroyed against the UK to claim their lone break point on Saturday, with a first-match score of 1,980,510 that would've easily been enough to beat China's Saturday performance by over 50,000 and secure victory, but this wasn’t Saturday anymore.
```

<div align="center" class="osu-md__paragraph">
    <video width="95%" controls>
        <source src="https://assets.ppy.sh/media/news/2026-04-18-twc26-sf-highlight-2.mp4" type="video/mp4" preload="none">
    </video>
</div>

> While China held a similar score to their first match, Italy weren't so lucky, with the brutal ending swinging the lead over to China in the dying moments. We were headed to our **first tiebreaker since the Round of 16**.

Reviewer 2: ```suggestion
While China held a similar score to their first match, Italy wasn't so lucky, with the brutal ending swinging the lead over to China in the dying moments. We were headed to our **first tiebreaker since the Round of 16**.
```

> An osu!taiko map of **[MWC7K 2026](/wiki/Tournaments/MWC/2026_7K)'s Grand Finals TB**, [Laur - SEV-26](https://osu.ppy.sh/beatmapsets/2534398#taiko/5604215), was all that could separate these two titans now. Nerves ran higher than ever, everyone seemed to miss on things they usually wouldn't, Italy held a slim lead for most of the first half, but eventually, China pulled away and wouldn't look back. With a brilliant recovery in the harder second half from China's strongest player, ::{ flag=CN }:: [superSSS](https://osu.ppy.sh/users/4315477), the ticket to Finals was punched in China's name, and Europe's last were eliminated from the osu!taiko World Cup. Italy go out in 7-8th place, after valiant efforts against many insanely strong teams, but China moves on to Finals to face a terrifying South Korea, fresh off a loss to Japan.

Reviewer 2: ```suggestion
An osu!taiko map of **[MWC 7K 2026](/wiki/Tournaments/MWC/2026_7K)'s Grand Finals TB**, [Laur - SEV-26](https://osu.ppy.sh/beatmapsets/2534398#taiko/5604215), was all that could separate these two titans now. Nerves ran higher than ever, everyone seemed to miss on things they usually wouldn't, Italy held a slim lead for most of the first half, but eventually, China pulled away and wouldn't look back. With a brilliant recovery in the harder second half from China's strongest player, ::{ flag=CN }:: [superSSS](https://osu.ppy.sh/users/4315477), the ticket to Finals was punched in China's name, and Europe's last were eliminated from the osu!taiko World Cup. Italy go out in 7–8th place, after valiant efforts against many insanely strong teams, but China moves on to Finals to face a terrifying South Korea, fresh off a loss to Japan.
```

> An osu!taiko map of **[MWC7K 2026](/wiki/Tournaments/MWC/2026_7K)**'s **Grand Finals TB**, [Laur - SEV-26](https://osu.ppy.sh/beatmapsets/2534398#taiko/5604215), was all that could separate these two titans now. Nerves ran higher than ever, everyone seemed to miss on things they usually wouldn't, Italy held a slim lead for most of the first half, but eventually, China pulled away and wouldn't look back. With a brilliant recovery in the harder second half from China's strongest player, ::{ flag=CN }:: [superSSS](https://osu.ppy.sh/users/4315477), the ticket to Finals was punched in China's name, and Europe's last were eliminated from the osu!taiko World Cup. Italy go out in 7-8th place, after valiant efforts against many insanely strong teams, but China moves on to Finals to face a terrifying South Korea, fresh off a loss to Japan.

Reviewer 3: ```suggestion
An osu!taiko map of **[MWC7K 2026](/wiki/Tournaments/MWC/2026_7K)'s Grand Finals TB**, [Laur - SEV-26](https://osu.ppy.sh/beatmapsets/2534398#taiko/5604215), was all that could separate these two titans now. Nerves ran higher than ever, everyone seemed to miss on things they usually wouldn't, Italy held a slim lead for most of the first half, but eventually, China pulled away and wouldn't look back. With a brilliant recovery in the harder second half from China's strongest player, ::{ flag=CN }:: [superSSS](https://osu.ppy.sh/users/4315477), the ticket to Finals was punched in China's name, and Europe's last were eliminated from the osu!taiko World Cup. Italy go out in 7-8th place, after valiant efforts against many insanely strong teams, but China moves on to Finals to face a terrifying South Korea, fresh off a loss to Japan.
```

Reviewer 3: the unbolded `'s` in the middle looks a lil weird imo

## Score highlights

> - The defending champions of the last 13 years, **Japan (#1)**, have looked relatively mortal so far this tournament, dropping an uncharacteristic 3 points across their first two matches leading into a [heavyweight bout](https://www.twitch.tv/videos/2745512622) with **South Korea (#5)**, but this week they proved that any rumours of their downfall have been greatly exaggerated. In a 6-1 win with 2 players on each map, Japan scored an unreal 14 out of 14 full combos, with only one score, one of two HDHR plays on the antimod [Aiyru - Elevator](https://osu.ppy.sh/beatmapsets/2534394#taiko/5604206) (FM2), even dipping below 99% accuracy!

Reviewer 2: consistent with how we capitalise mod names and mod categories

```suggestion
- The defending champions of the last 13 years, **Japan (#1)**, have looked relatively mortal so far this tournament, dropping an uncharacteristic 3 points across their first two matches leading into a [heavyweight bout](https://www.twitch.tv/videos/2745512622) with **South Korea (#5)**, but this week they proved that any rumours of their downfall have been greatly exaggerated. In a 6–1 win with 2 players on each map, Japan scored an unreal 14 out of 14 full combos, with only one score, one of two HDHR plays on the Anti Mod pick [Aiyru - Elevator](https://osu.ppy.sh/beatmapsets/2534394#taiko/5604206) (FM2), even dipping below 99% accuracy!
```

> - The other [winner's bracket semifinal](https://www.twitch.tv/videos/2745993870) between **Chile (#2)** and **Brazil (#3)** produced a week's worth of highlight reel moments on its own, from the Chilean mainstay ::{ flag=CL }:: [ulko](https://osu.ppy.sh/users/1263669) entering the match with a pair of SS plays on [Xyris - Terrablazer](https://osu.ppy.sh/beatmapsets/2534374#taiko/5604173) (HD2) and [Fallujah - Carved From Stone (2024 Remaster)](https://osu.ppy.sh/beatmapsets/2534385#taiko/5604193) (NM5), to Brazilian legend ::{ flag=BR }:: [HiroK](https://osu.ppy.sh/users/4050738)'s 99.82% HDHR FC on [fool - TO THE NEXT](https://osu.ppy.sh/beatmapsets/2534404#taiko/5604231) (FM3), to a tiny win margin of 301 on [Aiobahn feat. KOTOKO - INTERNET YAMERO](https://osu.ppy.sh/beatmapsets/2534395#taiko/5604207) (DT2), and after the match, even a 99.8%, 14x100 FC from ::{ flag=BR }:: [miwoo](https://osu.ppy.sh/users/12630336) on [Laur - SEV-26](https://osu.ppy.sh/beatmapsets/2534398#taiko/5604215) (TB) as it was played just for fun! At 6-3 for Chile, it wasn't quite as close as Italy vs China, but it's still worth a watch before Finals gets underway.

Reviewer 3: ```suggestion
- The other [winner's bracket semifinal](https://www.twitch.tv/videos/2745993870) between **Chile (#2)** and **Brazil (#3)** produced a week's worth of highlight reel moments on its own, from the Chilean mainstay ::{ flag=CL }:: [ulko](https://osu.ppy.sh/users/1263669) entering the match with a pair of SS plays on [Xyris - Terrablazer](https://osu.ppy.sh/beatmapsets/2534374#taiko/5604173) (HD2) and [Fallujah - Carved From Stone (2024 Remaster)](https://osu.ppy.sh/beatmapsets/2534385#taiko/5604193) (NM5), to Brazilian legend ::{ flag=BR }:: [HiroK](https://osu.ppy.sh/users/4050738)'s 99.82% HDHR FC on [fool - TO THE NEXT](https://osu.ppy.sh/beatmapsets/2534404#taiko/5604231) (FM3), to a tiny win margin of 301 on [Aiobahn feat. KOTOKO - INTERNET YAMERO](https://osu.ppy.sh/beatmapsets/2534395#taiko/5604207) (DT2), and after the match, even a 99.8%, 14x100 FC from ::{ flag=BR }:: [miwoo](https://osu.ppy.sh/users/12630336) on [Laur - SEV-26](https://osu.ppy.sh/beatmapsets/2534398#taiko/5604215) (TB) as it was played just for fun! At 6-3 for Chile, it wasn't quite as close as Italy vs China, but it's still worth a watch.
```

Reviewer 3: it's too late in the week to be talking about Before Finals now, this was only accurate when i initially wrote it up

> - The other [winner's bracket semifinal](https://www.twitch.tv/videos/2745993870) between **Chile (#2)** and **Brazil (#3)** produced a week's worth of highlight reel moments on its own, from the Chilean mainstay ::{ flag=CL }:: [ulko](https://osu.ppy.sh/users/1263669) entering the match with a pair of SS plays on [Xyris - Terrablazer](https://osu.ppy.sh/beatmapsets/2534374#taiko/5604173) (HD2) and [Fallujah - Carved From Stone (2024 Remaster)](https://osu.ppy.sh/beatmapsets/2534385#taiko/5604193) (NM5), to Brazilian legend ::{ flag=BR }:: [HiroK](https://osu.ppy.sh/users/4050738)'s 99.82% HDHR FC on [fool - TO THE NEXT](https://osu.ppy.sh/beatmapsets/2534404#taiko/5604231) (FM3), to a tiny win margin of 301 on [Aiobahn feat. KOTOKO - INTERNET YAMERO](https://osu.ppy.sh/beatmapsets/2534395#taiko/5604207) (DT2), and after the match, even a 99.8%, 14x100 FC from ::{ flag=BR }:: [miwoo](https://osu.ppy.sh/users/12630336) on [Laur - SEV-26](https://osu.ppy.sh/beatmapsets/2534398#taiko/5604215) (TB) as it was played just for fun! At 6-3 for Chile, it wasn't quite as close as Italy vs China, but it's still worth a watch before Finals gets underway.

Reviewer 3: ```suggestion
- The other [winner's bracket semifinal](https://www.twitch.tv/videos/2745993870) between **Chile (#2)** and **Brazil (#3)** produced a week's worth of highlight reel moments on its own, from the Chilean mainstay ::{ flag=CL }:: [ulko](https://osu.ppy.sh/users/1263669) entering the match with a pair of SS plays on [Xyris - Terrablazer](https://osu.ppy.sh/beatmapsets/2534374#taiko/5604173) (HD2) and [Fallujah - Carved From Stone (2024 Remaster)](https://osu.ppy.sh/beatmapsets/2534385#taiko/5604193) (NM5), to Brazilian legend ::{ flag=BR }:: [HiroK](https://osu.ppy.sh/users/4050738)'s 99.82% HDHR FC on [fool - TO THE NEXT](https://osu.ppy.sh/beatmapsets/2534404#taiko/5604231) (FM3), to a tiny win margin of 301 on [Aiobahn feat. KOTOKO - INTERNET YAMERO](https://osu.ppy.sh/beatmapsets/2534395#taiko/5604207) (DT2), and after the match, even a 99.8%, 14x100 FC from ::{ flag=BR }:: [miwoo](https://osu.ppy.sh/users/12630336) on [Laur - SEV-26](https://osu.ppy.sh/beatmapsets/2534398#taiko/5604215) (TB) as it was played just for fun! At 6-3 for Chile, who guaranteed their first ever podium finish with this win, it wasn't quite as close as Italy vs China, but it's still worth a watch before Finals gets underway.
```

Reviewer 3: big up my chilean friends

> - The other [winner's bracket semifinal](https://www.twitch.tv/videos/2745993870) between **Chile (#2)** and **Brazil (#3)** produced a week's worth of highlight reel moments on its own, from the Chilean mainstay ::{ flag=CL }:: [ulko](https://osu.ppy.sh/users/1263669) entering the match with a pair of SS plays on [Xyris - Terrablazer](https://osu.ppy.sh/beatmapsets/2534374#taiko/5604173) (HD2) and [Fallujah - Carved From Stone (2024 Remaster)](https://osu.ppy.sh/beatmapsets/2534385#taiko/5604193) (NM5), to Brazilian legend ::{ flag=BR }:: [HiroK](https://osu.ppy.sh/users/4050738)'s 99.82% HDHR FC on [fool - TO THE NEXT](https://osu.ppy.sh/beatmapsets/2534404#taiko/5604231) (FM3), to a tiny win margin of 301 on [Aiobahn feat. KOTOKO - INTERNET YAMERO](https://osu.ppy.sh/beatmapsets/2534395#taiko/5604207) (DT2), and after the match, even a 99.8%, 14x100 FC from ::{ flag=BR }:: [miwoo](https://osu.ppy.sh/users/12630336) on [Laur - SEV-26](https://osu.ppy.sh/beatmapsets/2534398#taiko/5604215) (TB) as it was played just for fun! At 6-3 for Chile, who guaranteed their first ever podium finish with this win, it wasn't quite as close as Italy vs China, but it's still worth a watch before Finals gets underway.

Reviewer 2: ```suggestion
- The other [winner's bracket Semifinals match](https://www.twitch.tv/videos/2745993870) between **Chile (#2)** and **Brazil (#3)** produced a week's worth of highlight reel moments on its own, from the Chilean mainstay ::{ flag=CL }:: [ulko](https://osu.ppy.sh/users/1263669) entering the match with a pair of SS plays on [Xyris - Terrablazer](https://osu.ppy.sh/beatmapsets/2534374#taiko/5604173) (HD2) and [Fallujah - Carved From Stone (2024 Remaster)](https://osu.ppy.sh/beatmapsets/2534385#taiko/5604193) (NM5), to Brazilian legend ::{ flag=BR }:: [HiroK](https://osu.ppy.sh/users/4050738)'s 99.82% HDHR FC on [fool - TO THE NEXT](https://osu.ppy.sh/beatmapsets/2534404#taiko/5604231) (FM3), and even a tiny win with a margin of 301 on [Aiobahn feat. KOTOKO - INTERNET YAMERO](https://osu.ppy.sh/beatmapsets/2534395#taiko/5604207) (DT2). Even after the match, there was even a 99.8%, 14x100 FC from ::{ flag=BR }:: [miwoo](https://osu.ppy.sh/users/12630336) on [Laur - SEV-26](https://osu.ppy.sh/beatmapsets/2534398#taiko/5604215) (TB) as it was played just for fun! At 6–3 for Chile, guaranteeing their first-ever podium finish with this win, it wasn't quite as close as Italy vs. China, but it's still worth a watch before the Finals get underway.
```
